### DISCUSSION TITLE
Welcome to codeceptjs-testrail Discussions!

### DISCUSSION THREAD

**Discussion by kobenguyent** [PeterNgTr/codeceptjs-testrail#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jayson-labarrete** [comment]
Hi, I am Jayson, 
I'm trying to use this in my test framework (codeceptjs + pupeteer)
I'm wondering if there is a sample github repo on how the config file was setup(so that I can reference from )? 

Should I copy paste the config in babelrc.json? or package.json? 
Thank you.

**Reply by kobenguyent** [reply]
Hi jayson, maybe this one is a good start for you https://github.com/PeterNgTr/codeceptjs-testrail/blob/master/test/config/mock.testrail.js

**Reply by jayson-labarrete** [reply]
Thanks @PeterNgTr I'll look into the link you shared.
